.claude/agents/maintainability-analyst.md

```
@@ -0,0 +1,327 @@
+
+**Minor issues:**
+- `auth.test.ts:89`: Flaky timing assertion (should use fake timers)
+
+### Error Handling (POOR)
+
+**Issues:**
+- **MAINT-005**: `src/auth.ts:78` - Empty catch block swallows database errors
+  ```typescript
+  try { await db.save(user); } catch (e) { /* ignore */ }
+  ```
+  - Impact: Silent failures, impossible to debug
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)